jobsiteScrape.py

```diff
@@ -104,10 +104,9 @@ def write_to_XLS(data_dictionary):
 
     # Do run time logging.
     try:
-        old_time_log = open('runlog.txt', 'r')
-        content = old_time_log.readlines()
-        old_time_log.close()
-        last_line = content[len(content) - 1].rstrip()
+        with open('runlog.txt', 'r') as old_time_log:

```

Comment:
This makes sense, yeah. Thanks for pointing this out, I hadn't learned about this, yet.